contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1419,13 +1419,13 @@ The following example shows a more complex use of user-chosen elements and presc
 
 [[backslash-for-long-code-line]]
 ==== Backslash for long lines of code
-For long lines of code that you want to break up among multiple lines, use a backslash to show the line break. For example:
+For long lines of code that you want to break up among multiple lines, use a backslash to show the line break and indent all lines under the first with two spaces. For example:
 
```

Comment:
Should we show the whole asciidoc block here? Such as

(source,terminal)
----
$ oc get endpoints --all-namespaces --template \
  '{{ range .items }}{{ .metadata.namespace }}:{{ .metadata.name }} \
  {{ range .subsets }}{{ range .addresses }}{{ .ip }} \
  {{ end }}{{ end }}{{ "\n" }}{{ end }}' | awk '/ 172\.30\./ { print $1 }'
----